bedrock/pocket/redirects.py

```diff
@@ -0,0 +1,7 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+from bedrock.redirects.util import redirect
+
+redirectpatterns = (redirect(r"^pocket-updates-signup/?$", "pocket.updates-signup"),)
```

Comment:
Waiting for @stevejalim to get help setting up the Pocket backend to handle redirects